test/unit/serializer.test.js

```diff
@@ -55,7 +55,11 @@ test('Long numerals', (t) => {
     `"positive": ${longPositive.toString()}, ` +
     `"array": [ ${longNegative.toString()}, ${longPositive.toString()} ], ` +
     `"negative": ${longNegative.toString()},` +
-    `"hardcoded": 102931203123987` +
+    `"hardcoded": 102931203123987,` +
+    `"a": "෴1111111111111111",` +
+    `"b": "߷1111111111111111",` +
+    `"c": "֍1111111111111111",` +
+    `"d": "෴֍߷1111111111111111"` +
     `}`;
```

Comment:
Don't you want to add the expected outputs like `t.equal(obj.hardcoded, 102931203123987);` on line 71?

Reply:
I will be adding similar ones in my other PR. I would recommend we hold off on these.

Reply:
Updated.